### ISSUE TITLE
media: 'skip to transcript' functionality doesn't work when JAWS is running

### ISSUE THREAD

**Issue by moloko** [adaptlearning/adapt_framework#2427]
### Subject of the issue/enhancement/features
If you select the 'skip to transcript' button in the media component when JAWS is running, focus is sent to the transcript button but pressing <kbd>Enter</kbd> has no effect

### Your environment
* FW v4.1.1 with adapt-contrib-media v4.0.1
* IE11 with Jaws 2019 (or Jaws 18)

Looks like adding a slight delay to the `onSkipToTranscript` function fixes this
